src/BUILD

```diff
@@ -1392,6 +1392,7 @@ cc_test(
                 "test/pythonnode_test.cpp",
                 "test/text_streamer_test.cpp",
                 "test/rerank_handler_test.cpp",
+                "test/embeddings_handler_test.cpp",
```

Comment:
Shouldn't it work when python is disabled as well?

Reply:
Additionally please make it as separate dependency of ovms_test - check openvino_remote_tensors or openvino_tests.